ReleaseNotes.md

```diff
@@ -11,6 +11,43 @@ additional questions or comments.
 
 Note : The agent version(s) below has dates (ciprod<mmddyyyy>), which indicate the agent build dates (not release dates)
 
+### 5/19/2022 -
+##### Version microsoft/oms:ciprod05192022 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod05192022 (linux)
+##### Version microsoft/oms:win-ciprod05192022 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod05192022 (windows)
+##### Code change log
+- Linux Agent
+  - PodReadyPercentage metric bug fix
+  - add cifs & fuse file systems to ignore list
+  - CA Cert Fix for Mariner Hosts in Air Gap
+- Windows Agent
+  - Ruby version upgrade from 2.6.5.1 to 2.7.5.1
+  - Added Support for Windows Server 2022
+  - Multi-Arch Image to support both Windows 2019 and Windows 2022
+- Common (Linux & Windows Agent)
+  - Telegraf version update from 1.20.3 to 1.22.2 to fix the vulnerabilitis
```

Comment:
Could we please add a note that disk usage metrics will no longer be collected for the paths "/mnt/containers" and "/mnt/docker" ?

Reply:
Updated!